advocacy_docs/partner_docs/CohesityDataProtectforPostgreSQL/04-ConfiguringCohesityDataProtectforPostgreSQL.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: 'Configuring Cohesity DataProtect for PostgreSQL'
+description: 'Walkthrough on configuring Cohesity DataProtect for PostgreSQL'
+---
+
+Implementing Cohesity DataProtect for PostgreSQL with EDB Postgres Advanced Server requires the following components:
+
+- EDB Postgres Advanced Server 
+- Cohesity Backup Appliance
+- Cohesity Linux Agent 
+- Cohesity PostgreSQL Agent 
+
+### Prerequisites
+
+- A running EDB Postgres Advanced Server instance
+- Cohesity Backup Appliance installed and running on VMware
+- Cohesity Linux and PostgreSQL agents installed on the same machine as your EDB Postgres Advanced Server database
+- The user account of the PostgreSQL instance must have database admin privileges
+- Login information for the Cohesity Web interface
+- Credentials that are used to access the EDB Postgres Advanced Server database
+- Java 1.8 or higher version is installed on the PostgreSQL nodes
+- PostgreSQL Agent scripts path, `/opt/cohesity/postgres/scripts`
```

Comment:
is this Cohesity agent?

Reply:
Yes so sorry somehow left out the word Cohesity.